sample_file.py

```
@@ -0,0 +1,48 @@
+        total += num
+    return total
+
+def divide(a, b):
+    return a / b
+
+def get_user_data(user_id):
+    query = "SELECT * FROM users WHERE id = " + user_id
+    print("Executing query: " + query)
+
+def write_to_file(data):
+    file = open("output.txt", "w")
```

Ruff

Use a context manager for opening files

(SIM115)